src/getHumanAge.test.js

```diff
@@ -3,8 +3,21 @@
 describe('getHumanAge', () => {
   const { getHumanAge } = require('./getHumanAge');
 
-  test('should be declared', () => {
-    expect(getHumanAge)
-      .toBeInstanceOf(Function);
+  test('should convert cat and dog years to human years correctly', () => {
+    expect(getHumanAge(15, 15)).toEqual([1, 1]);
+    expect(getHumanAge(24, 24)).toEqual([2, 2]);
+    expect(getHumanAge(30, 30)).toEqual([3, 3]);
+  });
+
+  test('should handle edge cases', () => {
```

Comment:
Separate test for each edge case is needed - tests should be atomic. Also more detailed information should be provided in the test name